R/utility.R

```diff
@@ -574,6 +660,268 @@ getXmlValues <- function(xmlResponse, xmlPath) {
   xml2::xml_text(xml2::xml_find_all(xmlResponse, xmlPath))
 }
 
-areShallowEqual <- function(a, b){
+areShallowEqual <- function(a, b) {
   !is.null(a) && !is.null(b) && a == b
 }
+
+validateClusterConfig <- function(clusterFilePath) {
```

Comment:
Please move all validation methods to a validationUtilities file.

Reply:
done